oss-internship-2020/sapi_lodepng/examples/main_unsandboxed.cc

```diff
@@ -0,0 +1,115 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <glog/logging.h>
+
+#include <iostream>
+
+#include "helpers.h"
+#include "lodepng.h"
+#include "sandboxed_api/sandbox2/util/fileops.h"
+
+void EncodeDecodeOneStep(const std::string& images_path) {
+  // Generate the values.
+  std::vector<uint8_t> image = GenerateValues();
+
+  // Encode the image.
+  const std::string filename = images_path + "/out_generated1.png";
```

Comment:
Any reason why you're not using sapi/sandbox2/util/path.h's JoinPath() here (and in other places)?